protocol/0072-SPPW-spam-protection-PoW.md

```diff
@@ -74,5 +74,54 @@ The initial hash-function used is SHA3 . To allow for a more fine-grained contro
+     - This is communicated to all wallets, and wallets use the new parameter for each transaction tied to a block with a height higher than the one in which the change happened.
+    - Transactions tied to such a block using the new difficulty are not rejected.
+    (<a name="0072-COSMICELEVATOR-011" href="#0072-COSMICELEVATOR-011">0072-COSMICELEVATOR-011</a>)
+
+
+- The parameter 'spam.pow.numberofTxPerBlock' is decrerased.  Verify that o
+        - The new parameter is communicated to and adapted by the wallets, i.e., if a user has too many transactions according to the new parameter, the wallet does not submit transactions with a too low
+ poW difficulty (either by submitting a PoW of higher difficulty, or by submitting the transactions later).
+        - The new parameter addapted for all blocks with a block height higher than the current one, i.e., a number of transactions that was allowed before and is no longer due to the new parameter is rejected (leaving the spam difficulty level constant)
+    (<a name="0072-COSMICELEVATOR-012" href="#0072-COSMICELEVATOR-012">0072-COSMICELEVATOR-012</a>)
+
+- The parameter 'spam.pow.numberofTxPerBlock' is increased. Verify that
+     - This is communicated to all wallets, and wallets use the new parameter for each transaction tied to a block with a height higher than the one in which the change happened. This means that a wallet
+ that had has a number of trnsactions exceeding the previous limit, but not the current one, does not increase the PoW difficulty or delay the transactions.
+    -  Accordingly, if a number of transactions is submitted with the level 1 PoW that would have exceeded the spam.pow.numberOfTXPerBlock pefore the change and not after, no transaction is rejected.
+    (<a name="0072-COSMICELEVATOR-013" href="#0072-COSMICELEVATOR-011">0072-COSMICELEVATOR-013</a>)
+
+
+- Repeat tests 008-013,  where in the same block,
+   - Spam.pow.difficulty is increased and spam.pow.increaseDifficulty is decreased, and spam.pow.numberOfTXPerBlock is increased.
+   - Spam.pow.difficulty is increased and spam.pow.increaseDifficulty is increased, and spam.pow.numberOfTXPerBlock is increased.
+   - Spam.pow.difficulty is decreased and spam.pow.increaseDifficulty is decreased, and spam.pow.numberOfTXPerBlock is increased.
+   - Spam.pow.difficulty is decreased and spam.pow.increaseDifficulty is increased, and spam.pow.numberOfTXPerBlock is increased.
+   - Spam.pow.difficulty is increased and spam.pow.increaseDifficulty is decreased, and spam.pow.numberOfTXPerBlock is decreased.
+   - Spam.pow.difficulty is increased and spam.pow.increaseDifficulty is increased, and spam.pow.numberOfTXPerBlock is decreased.
+   - Spam.pow.difficulty is decreased and spam.pow.increaseDifficulty is decreased, and spam.pow.numberOfTXPerBlock is decreased.
+   - Spam.pow.difficulty is decreased and spam.pow.increaseDifficulty is increased, and spam.pow.numberOfTXPerBlock is decreased.
```

Comment:
We can probably cut these down given that if the enabling of `increaseDifficulty` takes effect the values of `difficulty` and `numberOfTXPerBlock` won't matter